torch/csrc/jit/codegen/cuda/index_compute.cpp

```diff
@@ -249,6 +250,11 @@ void IndexCompute::handle(Split* split) {
   } else {
     index_map_[in_id] =
         kir::addExpr(kir::mulExpr(outer_ind, getExtent(inner_id)), inner_ind);
+    if (extent_map_.find(outer_id) != extent_map_.end() ||
```

Comment:
can you please add a comment explaining the logic?

Reply:
I actually don't know. I guess `extent_map_[in_id]` should be updated with the multiplication without the conditional. I'll ask Christian when he's back.